en/guides/manage-devices/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Introduction
+table_of_contents: true
+---
+
+# Introduction
+
+On some systems it is appropriate to have a system user. For example, with a system user, one can ssh to the device and run commands.
+
+By default on first boot, stock Core images run a program called console-conf. If the user has a keyboard and monitor attached to the device, they can use console-conf to configure the network and add a user account. This user account can then be used to ssh to the device for various management purposes.
+
+Some systems suppress console-conf,  and a user is not created by default. In these cases, one may add a system-user if one has the information and authority needed to do so for the particular system. This is done by inserting a USB drive with a special file (named `auto-import.assert`) in the root directory. Snapd imports the assertions found in it, and, if the assertions are valid for the given system, the system-user is created. After this, you can log into the device (locally or over ssh) using the username and password defined.
+
+In general, you cannot add a system-user to a system that is already "managed". A system is managed if a user was already created by console-conf, or if a system-user was created through the auto-import mechanism described here.
+
+Whether you can add a system-user to an unmanaged system depends on the model assertion that was used to create the image. It also depends on whether you have an authorized snapcraft key on your local system. For a discussion of keys, see [Ubuntu Core Images](../build-device/image-building.md).
+
+## Model assertions that allow adding a system user
+
+### The simplest case is where you did everything:
+
+* You created a snapcraft key and it is registered to your SSO
+* You created a model assertion
+* The model assertion has the `authority-id` set to your SSO account ID
+* The model assertion has the `brand-id` set to your SSO account ID
+* You signed the model assertion using the key mentioned above
+* You built the image using that model assertion
+
+The model assertion (original JSON) may look like this:
+
+```
+{
+  "type": "model",
+  "authority-id": "THE-SSO-ACCOUNT-ID",
+  "brand-id": "THE-SSO-ACCOUNT-ID",
+  "series": "16",
+  "model": "mymodel",
+  "architecture": "armhf",
+  "kernel": "pi2-kernel",
+  "gadget": "cm3",
+  "timestamp": "2017-07-11T15:55:59+00:00"
+}
+```
+
+Note that this model assertion does not have a `system-user-authority` key. When a model assertion lacks this key, a system-user assertion can only be signed by a key registered to the SSO account specified by the `brand-id` field. But, in this case, you do have the key needed to sign the system-user assertion.
```

Comment:
`system-user-authority` key
It may be clearer to a "field" rather than a "key", otherwise it sounds as though you are referring to a GPG key i.e. a signing key.